### ISSUE TITLE
Support non-square grids

### ISSUE THREAD

**Issue by deepnight** [deepnight/led#18]

**Comment by Geokureli** [comment]
Just wanted to say that this would be pretty awesome

**Comment by daimahou** [comment]
Any update on this ? Would be great to have support for this ? If you'd be willing to give me some instruction, I could probably go in and contribute on that task if need be.

**Comment by SamuelRabreau** [comment]
Indeed, this is what made me chose Tiled over LDtk. No support for hex or even rectangular tiles.
